src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/DialogRequestedEventArgs.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class contains the requests dialog from vc-daemon.
+    /// </summary>
+    public class DialogRequestedEventArgs : EventArgs
+    {
+        internal DialogRequestedEventArgs(int pid, string dispTextString, string uttTextString, bool continuous)
+        {
+            Pid = pid;
+            DisplayText = dispTextString;
+            UtterenceText = uttTextString;
+            IsContinuous = continuous;
+        }
+
+        /// <summary>
+        /// The process id of VC client to request dialog
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Pid
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// The display text requested to be displayed
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string DisplayText
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// The utterence text requested to be spoken
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string UtterenceText
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Continue dialog session
```

Comment:
I still do not understand.
What means of `Continue dialog session`?

Reply:
`It is indicate that this dialog request is a continuation of the previous dialog`
Is it right?

Reply:
Would you please update description?

Reply:
Updated.